cli/cmd/version/version.go

```diff
@@ -2,49 +2,32 @@ package version
 
 import (
 	"fmt"
-	"strings"
-)
+	"sync"
 
-var (
-	// The git commit that was compiled. These will be filled in by the compiler.
-	GitCommit   string
-	GitDescribe string
-
-	// The main version number that is being run at the moment.
-	//
-	// Version must conform to the format expected by
-	// github.com/hashicorp/go-version for tests to work.
-	Version = "0.0.1"
-
-	// A pre-release marker for the version. If this is "" (empty string)
-	// then it means that it is a final release. Otherwise, this is a pre-release
-	// such as "dev" (in development), "beta", "rc1", etc.
-	VersionPrerelease = ""
+	"github.com/hashicorp/consul-k8s/cli/cmd/common"
 )
 
-// GetHumanVersion composes the parts of the version in a way that's suitable
-// for displaying to humans.
-func GetHumanVersion() string {
-	version := Version
-	if GitDescribe != "" {
-		version = GitDescribe
-	}
-
-	release := VersionPrerelease
-	if GitDescribe == "" && release == "" {
-		release = "dev"
-	}
-
-	if release != "" {
-		if !strings.HasSuffix(version, "-"+release) {
-			// if we tagged a prerelease version then the release is in the version already
-			version += fmt.Sprintf("-%s", release)
-		}
-		if GitCommit != "" {
-			version += fmt.Sprintf(" (%s)", GitCommit)
-		}
-	}
-
-	// Strip off any single quotes added by the git information.
-	return strings.Replace(version, "'", "", -1)
+type Command struct {
+	*common.BaseCommand
+
+	Version string
+	once    sync.Once
+}
+
+func (c *Command) init() {
+	c.Init()
+}
+
+func (c *Command) Run(_ []string) int {
+	c.once.Do(c.init)
+	c.UI.Output(fmt.Sprintf("consul-k8s %s", c.Version))
+	return 0
+}
+
+func (c *Command) Synopsis() string {
+	return "Prints the version."
+}
+
+func (c *Command) Help() string {
+	return ""
```

Comment:
Should there be something returned here?

Reply:
yeah good call, I copied https://github.com/hashicorp/consul/blob/main/command/version/version.go#L90